scripts/docker/build.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+CURDIR=$(dirname $(realpath "$0"))
+WORKDIR=$(realpath ${CURDIR}/../..)
+DOCKER_FILE=$WORKDIR/scripts/docker/opensuse-rust.docker
+
+IMAGE_NAME=opensuse-rust
+CONTAINER_NAME=coconut-build
+
+# Command line arguments
+ARG_REUSE=0
+
+####
+#### Help function
+####
+Help()
+{
+    echo "Build the SVSM within a container."
+    echo
+    echo "Syntax: $0 [-r|v|h]"
+    echo "options:"
+    echo "r    Reuse the ${CONTAINER_NAME} to build the SVSM"
+    echo "v    Verbose mode"
+    echo "h    Print this help"
+}
+
+####
+#### ParseOptions function
+####
+ParseOptions()
+{
+    while getopts "rvh" option; do
+        case $option in
+            r) # Reuse option
+                ARG_REUSE=1
+                ;;
+            v) # Verbose mode
+                set -x
+                ;;
+            h) # Display help
+                Help
+                exit
+                ;;
+            \?) # Invalid option
+                echo -e "ERR: Invalid option\n"
+                Help
+                exit
+                ;;
+        esac
+    done
+}
+
+####
+#### Reuse container to build the SVSM
+####
+BuildSvsmReuse()
+{
+    CONTAINER_ID=$(docker ps -q -f name=${CONTAINER_NAME})
+
+    # Create and start the the container if it's not running
+    if [ -z ${CONTAINER_ID} ] ; then
+
+        CONTAINER_ID=$(docker ps -q -f name=coconut-build)
```

Comment:
Should we add `-a` here?
We can also use `${CONTAINER_NAME}`

```suggestion
        CONTAINER_ID=$(docker ps -q -a -f name=${CONTAINER_NAME})
```

Reply:
Followup patch sent: https://github.com/coconut-svsm/svsm/pull/83